CanvasColorSpaceProposal.md

```diff
@@ -198,9 +198,134 @@ The ``getImageData`` method is responsible for converting the data from the canv
 ## Examples
 
 ### Selecting the best color space match for the user agent's display device
-<pre>
-var colorSpace = window.matchMedia("(color-gamut: p3)").matches ? "display-p3" : "srgb";
-</pre>
+
+This example selects a wide color gamut canvas only if the underlying display has a P3 gamut or larger.
+
+```html
+  // Note that the gamut is named p3, but the color space is 'display-p3'.
+  var matchingColorSpace = window.matchMedia(
```

Comment:
Prefer "let" or "const" instead of "var" throughout.

Reply:
Changed to let in all instances.

Reply:
Thanks, looks good.